frida_mode/src/main.c

```diff
@@ -220,7 +220,7 @@ __attribute__((visibility("default"))) void afl_frida_start(void) {
   persistent_config();
   prefetch_config();
   ranges_config();
-  #if !defined(__ANDROID__)
+  #if defined(__linux__) && !defined(__ANDROID__)
   seccomp_config();
```

Comment:
I don't think we should need any conditional compilation in the rest of AFL++ FRIDA mode, I think it should be possible to isolate all the conditional compilation to the `seccomp` portion like [this](https://github.com/AFLplusplus/AFLplusplus/blob/ba3c7bfe40f9b17a691958e3525828385127ad25/frida_mode/src/seccomp/seccomp.c#L12).